### PR TITLE
Currency & settings

### PR DESCRIPTION
**Pull request by JoshuaSlui**
This PR adds the bare basics for currency commands and settings. Currently, this PR offers:
- Disabling/Enabling currency per guild
- Seeing an overview of settings
- Checking your balance

I plan to do the following after this PR is pushed, as to not put too much load on this PR:
1. Add more currency commands
2. Add functionality for disabling social commands (per guild)

### CODE REVIEW

**Comment by JoshuaSlui** [comment]
I also intend to minimize some of the non-DRY code by making additional functions.

File: cogs/currency.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from discord import slash_command
+from discord.ext import commands
+import utils
+from utils import *
```
Comment (ToothyDev) [inline cogs/currency.py]: I would suggest you simply do `import utils` in order to not clutter the namespace by doing `from utils import *`

File: cogs/settings.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+from discord.commands import SlashCommandGroup
+from discord.ext import commands
+import utils
+from utils import *
```
Comment (ToothyDev) [inline cogs/settings.py]: Duplicate imports here as well

File: utils.py [hunk 3]
```diff
@@ -99,3 +116,10 @@ async def apireq(url):
         async with cs.get(url) as r:
             js = await r.json()
             return js
+
+async def mysql_login():
+    return mysql.connect(
+        host=config.database['host'],
+        user=config.database['user'],
+        password=config.database['password'],
+        database=config.database['database'])
```
Comment (ToothyDev) [inline utils.py]: Missing newline at end of file

**Review by ToothyDev — CHANGES_REQUESTED** [review]

**Review by ToothyDev — APPROVED** [review]

**Pull request merged**